_includes/workshops/neurips/neurips2019.html

```diff
@@ -0,0 +1,84 @@
+<div class="container">
+    <div class="blog-header">
+      <h1 class="blog-title">Accepted Papers</h1>
+    </div>
+
+    <div class="blog-header">
+      <h2 class="blog-title">Extended Abstracts</h2>
```

Comment:
For this workshop, the papers were just 'Abstracts' please remove the 'Extended' from the title.